### ISSUE TITLE
Changing theme for main site alters admin default theme

### ISSUE THREAD

**Issue by Mandyrobin** [tooblue/mytcg-f3#15]
Here's something I noticed after I started creating my own theme. When I switch to my new theme for the main website and I go to the admin panel, which still has the default theme selected, it looks like this:

![adminthemeerror](https://cloud.githubusercontent.com/assets/1059043/15752673/31efe332-28bd-11e6-8517-c4defecfb869.jpg)

Not sure what's causing the styling mixup since I didn't alter anything in the admin theme itself. It goes back to normal when I change the main site theme back to default.

**Comment by myubi** [comment]
Hey Mandy! :)
I have been helping around with MyTCF-f3 so I may be able to help you. Can I create an account in your TCG and be made an admin so I can do some checking on what have been breaking? 

Also, I have a new version of the dashboard with all the features working if it interests you! Bloo didn't have time to revise yet, so it may have some flaws, but I am up to sharing it! :)

**Comment by Mandyrobin** [comment]
Sure. I set the main theme back to default so you can sign up. The address is http://holywar.crystalline-ice.net

**Comment by myubi** [comment]
User created! :D I'm myubi, hahaha. 

Also, you will need this fix so users don't get errors when joining: https://github.com/filler00/mytcg-f3/commit/f8019362d086562f4837d1e6643161813f1514a8

**Comment by Mandyrobin** [comment]
Alright. Set you as admin (I think) and changed the theme back to my custom one. The admin panel address is the same as the config file default.

Yeah, that joining error thing. Gonna work on that eventually. :P

**Comment by myubi** [comment]
Yup, I'm admin! Will take a look a bit later today, will give you a proper reply asap.

**Comment by Mandyrobin** [comment]
Awesome, thanks! I'll keep working my pages and stuff in the meantime. Also, I would totally love to see the new functional dashboard. I really want to work with this script so the more the merrier! :D I also can't wait to see the news, randomizer and all the other awesome stuff.

**Comment by myubi** [comment]
Okay, actually, found it quite quickly, hahaha. Will have to work on a fix later though, but it's simple.

When you change the theme you are using, it changes the theme name for all css urls. 
So, now, all stylesheets are trying to be found inside your new theme folder, even the admin css. So, it's trying to find the admin-style.css on your holywar theme folder, which is not finding anything and then breaking all the styling that bloo did. 
<link rel="stylesheet" href="/minify/css?files=app/themes/holywar/assets/css/admin-style.css">
If you try to change the stylesheet url to the default theme, it will load the admin-style.css and fix everything.
<link rel="stylesheet" href="/minify/css?files=app/themes/default/assets/css/admin-style.css">

For a quick fix just copy all the files of the admin css on your new theme. I will work on a proper fix later.

**Comment by Mandyrobin** [comment]
Okie doke. Thanks. :)
